samples/wifi/sta/README.rst

```diff
@@ -129,6 +134,37 @@ Testing
       <inf> sta: ==================
       <inf> sta: State: DISCONNECTED
 
+Power management testing
+************************
+
+This sample can be used to measure current consumption of both the nRF5340 SoC and nRF7002 DK independently using two separate Power Profiler Kits (PPK).
+The nRF5340 SoC is connected to the PPK1 and the nRF7002 DK is connected to the PPK2.
+
+Hardware modifications
+======================
+
+To measure the current consumption of the nRF5340 SoC, complete the following steps:
+
+ * Remove jumper on **P22** (VDD jumper).
+ * Cut solder bridge **SB16**.
+ * Make a short on solder bridge **SB17**.
+ * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.
+    You can use **P21** pin **1** mentioned as **GND** (-).
+ * Connect the **Vout** on PPK2 to **P22** pin **1** on the nRF7002 DK.
+
+To measure the current consumption of the nRF7002 DK, complete the following steps:
+
+ * Remove jumper on **P23** (VBAT jumper).
+ * Connect **GND** on PPK2 kit to **GND** on the nRF7002 DK.
+    You can use **P21** pin **1** mentioned as  **GND** (-).
+ * Connect the **Vout** on PPK2 to **P23** pin **1** on the nRF7002 DK.
+
+PPKII Usage and measurement
```

Comment:
```suggestion
PPK2 usage and measurement
```

Reply:
This was intentional, don't want to confuse PPK2 with second PPK, this is PPK HW version II (As per doc as well its II)